storage/objects/main.go

```diff
@@ -0,0 +1,196 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Sample objects creates, list, deletes objects and runs
```

Comment:
Sample listobjects creates an object, lists objects from a bucket, and deletes that object.